package.json

```diff
@@ -6,10 +6,13 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "bootstrap": "^4.6.0",
     "react": "^17.0.2",
+    "react-bootstrap": "^1.5.2",
```

Comment:
why we are using this library, as we are already using "bootstrap" ?

Reply:
by the way, we can directly use CDN for bootstrap instead of the library (bootstrap and react-bootstrap) to avoid this!